tools/importer-rest-api-specs/components/dataapigeneratorjson/models/terraform_resource_schema.go

```diff
@@ -0,0 +1,40 @@
+package models
+
+import "github.com/hashicorp/pandora/tools/sdk/resourcemanager"
+
+// SchemaModel holds information about all the fields associated with a Terraform Resource
+type SchemaModel struct {
+	// Fields describes all the fields associated with a Terraform Resource
+	Fields []SchemaField `json:"Fields"`
+}
+
+// SchemaField describes information about an attribute that will be mapped into the Terraform Schema of a resource
+type SchemaField struct {
+	// HclName specifies the snaked cased attribute name (e.g. resource_group_name)
+	HclName string `json:"HclName"`
+
+	// Documentation describes what this attribute is
+	Documentation *string `json:"Documentation,omitempty"`
```

Comment:
this should be a struct with a field `Markdown` inside it, to give us the means of adding other fields (e.g. a list of uris to examples) in here in the future